README.md

```diff
@@ -0,0 +1,126 @@
+ghd deploy
+```
+
+`ghd` will ask for some inputs:
+```shell
+Ref [c5ff55e7e7d6a1f1d33b58bfb4efc00893cb8d08]: 
+Environment (dev, test, live): dev
+Transient [y/N]: 
+Production [y/N]: 
+Check constraints [Y/n]: y
+moneymeets/demo-repository@c5ff55e7e7d6a1f1d33b58bfb4efc00893cb8d08 (live/v1-2-gc5ff55e) will be deployed to dev
+  transient          no
+  production         no
+  required contexts  all
+  description        Github: Here is your commit message from c5ff55e7e7d6a1f1d33b58bfb4efc00893cb8d08
+Start deployment? [y/N]: y
+```
+
+The output looks like this:
+```shell
+Working in moneymeets/demo-repository
+Creating deployment
+::set-output name=deployment_id::123456789
+```
+
+#### Deployment check
+You can check the triggered deployment with the inspect command by specifying the deployment id:
+```shell
+ghd inspect 123456789
+```
+
+Output:
+```shell
+Working in moneymeets/demo-repository
+state        environment    creator              created               description
+-----------  -------------  -------------------  --------------------  -------------
+success      dev            github-actions[bot]  2020-02-06T08:32:52Z
+in_progress  dev            github-actions[bot]  2020-02-06T08:32:48Z
+```
+
+#### Deployment history
+If you need only the history of deployments, use:
+```shell
+ghd list
+```
+
+For detailed information use `-v`, you can limit you history with `-l`
+```shell
+ghd list -l 7 -v
+```
+
+Output:
+```shell
+Working in moneymeets/demo-repository
+
+       id  ref      task    environment    creator      created               status_changed        transient    production    state    description
+---------  -------  ------  -------------  -----------  --------------------  --------------------  -----------  ------------  -------  ----------------
+123456789  0695e4c  deploy  dev            demo-user2   2020-02-06T08:32:10Z  2020-02-06T08:32:52Z  no           no            success  Deployed via GHD
+223456789  0695e4c  deploy  dev            demo-user1   2020-02-06T08:24:18Z  2020-02-06T08:25:03Z  no           no            success  Deployed via GHD
+323456789  0695e4c  deploy  dev            demo-user1   2020-02-06T08:00:36Z  unknown               no           no            unknown  Deployed via GHD
+423456789  40bd8c8  deploy  dev            demo-user2   2020-01-24T09:00:24Z  2020-01-24T09:01:10Z  no           no            success  Deployed via GHD
+523456789  40bd8c8  deploy  dev            demo-user1   2020-01-24T08:58:15Z  2020-01-24T08:59:00Z  no           no            success  Deployed via GHD
+623456789  8857621  deploy  dev            demo-user2   2020-01-23T08:27:47Z  2020-01-23T08:33:22Z  no           no            success  Deployed via GHD
+723456789  74d43f0  deploy  dev            demo-user1   2020-01-23T08:07:28Z  2020-01-23T08:12:51Z  no           no            failure  Deployed via GHD
+```
+
+#### Disable environment
+You can disable an environment, this will hide the environment from GitHub UI.
```

Comment:
```suggestion
You can disable an environment, this will hide the environment from the GitHub UI. Note that this doesn't affect the number of environments shown on the main repository page.
```